### DISCUSSION TITLE
Emulator

### DISCUSSION THREAD

**Discussion by CVS67** [bitfocus/companion#2792] [Q&A] [answered]
Hello everybody,
I don't quite understand what emulator means! Is it possible to emulate functional buttons on an iPad or iPhone?, or is this just a preview of what you'll see on a stream deck?
Thanks for reading and for any reply.

**Comment by dnmeid** [comment, marked as answer]
If you look at the Companion admin webinterface, you'll see a sidebar on the left side. The last point is "Getting Started". There you'll find all information about the emulator and all other aspects of Companion.
And the short answer: yes, you can emulate working buttons on a tablet.
